blocks/inspector-controls/radio-control/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from 'components';
+import { isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from './../base-control';
+import './style.scss';
+
+function RadioControl( { label, selected, instanceId, onChange, options = [] } ) {
+	const id = 'inspector-radio-control-' + instanceId;
+	const onChangeValue = ( event ) => onChange( event.target.value );
+
+	return ! isEmpty( options ) && (
+		<BaseControl label={ label } id={ id }>
+			<div className="blocks-radio-control">
+				{ options.map( ( option, i ) =>
+					<label key={ option.value } htmlFor={ ( id + '-' + i ) }>
```

Comment:
I think for accessibility reasons, we're avoiding nesting inputs in labels and always using `htmlFor` as an alternative.